src/pages/term/nextBackButtons/nextBackButtons.module.sass

```
@@ -0,0 +1,4 @@
+.button-group
+    display: flex
+    gap: 2em
+    justify-content: end
```

Prettier doesn't work?
```suggestion
    justify-content: end

```